Jenkinsfile

```diff
@@ -237,11 +239,9 @@ pipeline {
               }
               when {
                 beforeAgent true
-                allOf {
```

Comment:
Otherwise a github comment will never force the `benchmark` to get triggered since `params.bench_ci`= false by default